### PR TITLE
Add lab-config files for as7512 switch. Changes password to onie-image defaults.

### PR DESCRIPTION
**Pull request by NStetskovych**

### CODE REVIEW

**Comment by NStetskovych** [comment]
@lguohan please review

**Comment by garyachy** [comment]
@qiluo-msft and @stcheng please review

File: ansible/group_vars/sonic/vars [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ansible_ssh_user: admin
-ansible_ssh_pass: password
```
Comment (qiluo-msft) [inline ansible/group_vars/sonic/vars]: >password [](start = 18, length = 8)

No need to change this file. It is just an example.

File: ansible/group_vars/sonic/vars [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
-ansible_become_pass: password
+ansible_ssh_pass: admin
+ansible_become_pass: admin
```
Comment (qiluo-msft) [inline ansible/group_vars/sonic/vars]: Thanks for catching this. I don't like the default password in sonic-buildimages. Could you also change it to "YourPaSsWoRd" at both places?

File: ansible/minigraph/switch4.xml [hunk 3]
```diff
@@ -0,0 +1,916 @@
+<DeviceMiniGraph xmlns:i="http://www.w3.org/2001/XMLSchema-instance" xmlns="Microsoft.Search.Autopilot.Evolution">
```
Comment (qiluo-msft) [inline ansible/minigraph/switch4.xml]: Could you summarize what are special in this minigraph comared with others? If no, you may reuse existing one.

Comment (lguohan) [inline ansible/minigraph/switch4.xml]: I prefer not to introduce a new minigraph unless there is a reason.

Comment (NStetskovych) [inline ansible/minigraph/switch4.xml]: @lguohan @qiluo-msft as far as I understand minigraph_facts use {{ inventory_hostname }} and use 'minigraph/{}.xml' to parse data i.e -mgmt ip. Deployment fails for me if I have switch4 in inventory but do not have corresponding xml under minigraph folder. 
Please correct me if I wrong.

File: ansible/group_vars/sonic/vars (hunk 2 above)
Comment (qiluo-msft) [inline ansible/group_vars/sonic/vars]: I submitted a PR: [https://github.com/Azure/sonic-buildimage/pull/197/files](https://github.com/Azure/sonic-buildimage/pull/197/files). Please wait after that.

---
In reply to: [96720233](https://github.com/Azure/sonic-mgmt/pull/80#discussion_r96720233) [](ancestors = 96720233)

**Comment by NStetskovych** [comment]
@lguohan removed minigraph. Just default passwd here.
